daemon/daemon.go

```diff
@@ -15,6 +15,9 @@ import (
 	"time"
 
 	"github.com/docker/libcontainer/label"
+	"github.com/docker/libnetwork"
+	"github.com/docker/libnetwork/pkg/netlabel"
```

Comment:
@stevvooe @mrjana godoc crawler is buggy and I fixed it for now and now it is indexing the directories properly : http://godoc.org/github.com/docker/libnetwork.
Yes, we can add good package documentation for all the packages. But that can be done later.